SETUP/document_database.php

```diff
@@ -0,0 +1,273 @@
+<?php
+
+$relPath='../pinc/';
+include_once($relPath.'base.inc');
+include_once($relPath.'misc.inc');
+include_once($relPath . 'TableDocumentation.inc');
+
+// First is the script name, the second is the operation name
+$number_of_arguments = $argc - 2;
+
+if ($argc < 2) {
+    echo "No operation was chosen.\n";
+    echo "Supported operations are: generate, verify, update\n";
+
+    exit(1);
+}
+
+// generate <directory_path> <table_name or all>
+if ($argv[1] === 'generate') {
+    if ($argc !== 4) {
+        echo "Operation generate requires 2 arguments, $number_of_arguments were given.\n";
+        echo "Supported syntax for generate command is 'generate <directory path> <table name or all>'.\n";
+
+        exit(1);
+    }
+
+    $directory_path = $argv[2];
```

Comment:
99% of the time we will want to run this against a directory that exists in the code checkout. It would be useful to have this default to that directory. Perhaps swap this and the `table` arguments so that `directory_path` is last.

Consider a directory name of `SETUP/dbdocs/` (so `$relPath/../SETUP/dbdocs/`).

Reply:
Done in c07c32a.